segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceConfig.java

```diff
@@ -35,6 +35,9 @@
     public static final Property<Integer> ZK_RETRY_SLEEP_MS = Property.named("zkRetrySleepMs", 5000);
     public static final Property<Integer> ZK_RETRY_COUNT = Property.named("zkRetryCount", 5);
     public static final Property<String> CLUSTER_NAME = Property.named("clusterName", "pravega-cluster");
+    public static final Property<String> STORAGE_IMPLEMENTATION = Property.named("storageImplementation",
```

Comment:
That is how the default was before. The configuration defaults to in-memory, and it should be set to whatever it should be in the deployment. That way it forces you to properly setup your config vs just running with some unconfigured HDFS